.github/workflows/release.yml

```diff
@@ -31,12 +31,12 @@ jobs:
         mvn -B package --file pom.xml
     - name: Set version env variable
       run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
-    - name: Get release details
-      run: |
-        export RELEASE_DETAILS="$(awk -vN=2 'n<N;/^# /{++n}' CHANGELOG.md)" 
-        export RELEASE_DETAILS="$(sed '${/^# /d;}' <<< "$RELEASE_DETAILS")"
-        touch RELEASE_DETAILS.md
-        echo "$RELEASE_DETAILS" > RELEASE_DETAILS.md
+    - name: 'Install GPG Secret Key'
+      id: import_gpg
+      uses: crazy-max/ghaction-import-gpg@v4
```

Comment:
@hsuamz are there any security implications with using this third-party gh action? One thing I'm unclear on is a lot of github actions published by others are not officially certified by GitHub but I imagine are still widely used, so I'm not sure if there's security concerns there are not. See the text on the right of [this page](https://github.com/marketplace/actions/import-gpg)

Reply:
We are good here. The secret is still secure on GitHub and used within the workflow machine and then cleaned up. Access is only limited to the workflow without public access.